sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosDiagnosticsTest.java

```diff
@@ -227,7 +227,7 @@ public void gatewayDiagnostics() throws Exception {
         InternalObjectNode internalObjectNode = getInternalObjectNode();
```

Comment:
I think we would need to "merge" in the changes related to this PR (the diagnostic enhancement PR for channel acquisition threshold violation) - https://github.com/Azure/azure-sdk-for-java/pull/38416/files#diff-b1526bced1f05d3901ffc6b338ebcc986b8ae37f1238f57f2d8b1579509fbd02

Reply:
Same for `RntbdTransportClientTest`.

Reply:
Oh sorry - are we porting this PR - https://github.com/Azure/azure-sdk-for-java/pull/38416 - it shows up in the changelog of the backport PR. If not, please ignore / resolve my comments.

Reply:
Yes - the PR with the channel acquisition changes was cherry-picked intentionally.

Reply:
Got it - in that case, I don't see the changes from (#38416) in the hotfix PR.

Reply:
Good catch :-) Something off when I merged the conflicts I guess. Added the cherry-pick for the missing PR now.